react/__tests__/components/__snapshots__/InfoCard.test.js.snap

```diff
@@ -173,7 +173,7 @@ exports[`<InfoCard /> should render with image and text side by side 1`] = `
       class="infoCardImageContainer w-50-ns"
     >
       <img
-        alt=""
+        alt="CLIQUE HERE"
```

Comment:
```suggestion
        alt="CLICK HERE"
```

Reply:
> 

@danzanzini Hi, sorry for the delay and I was off, could you check now?